resources/views/items/form/_entry.blade.php

```diff
@@ -25,6 +25,14 @@
         @include('cruds.fields.character', ['quickCreator' => true])
     </div>
 </div>
+<div class="row">
```

Comment:
To be consistent with the rest of the app, the parent field should be placed higher up in the form. please place it after type.